packages/client/components/MeetingRoot.tsx

```diff
@@ -7,7 +7,6 @@ import renderQuery from '../utils/relay/renderQuery'
 import MeetingSelector from './MeetingSelector'
 import SetAppLocationMutation from '~/mutations/SetAppLocationMutation'
 
-// Changing the name here requires a change to getLastSeenAtURL.ts
```

Comment:
We have a few of these comments in the app but it looks like the `getLastSeenAtURL.ts` isn't being used anywhere so I've removed these comments and the file